src/python/pants/backend/python/tasks/pytest_run.py

```diff
@@ -482,6 +483,21 @@ def _test_runner(self, workdirs, test_targets, sources_map):
                                           pytest_binary.pex) as coverage_args:
         yield pytest_binary, [conftest] + coverage_args, get_pytest_rootdir
 
+  def _constrain_pytest_interpreter_search_path(self):
+    """Return an environment for invoking a pex which ensures the use of the selected interpreter.
+
+    When creating the merged pytest pex, we already have an interpreter, and we only invoke that pex
+    within a pants run, so we can be sure the selected interpreter will be available. Constraining
+    the interpreter search path at pex runtime ensures that any resolved requirements will be
+    compatible with the interpreter being used to invoke the merged pytest pex.
+    """
+    pytest_prep_binary_product = self.context.products.get_data(PytestPrep.PytestBinary)
```

Comment:
You're using java words like I would tend to - this is surprising. Just `pytest_binary` seems fine since no line below care this is a product - just this line here where the RHS makes that clear anyway.

Reply:
I'll make this fix, and will keep this in mind. I've been trying to avoid abbreviations in variable names, but writing out irrelevant information as you've described is also not ideal.